requirements.txt

```diff
@@ -3,4 +3,5 @@ numpy
 networkx
 matplotlib
 cython
-scipy
\ No newline at end of file
+scipy
+loky
```

Comment:
does https://stackoverflow.com/a/65459316 work to define a platform-specific requirement?

Reply:
thanks for looking! lets leave it and give everyone a loky install. would be nice to get rid of RDKit and be able to be pip-installable – potentially something for the future